src/javascript/app/pages/user/metatrader/metatrader.config.js

```diff
@@ -133,12 +133,12 @@ const MetaTraderConfig = (() => {
                     };
 
                     const has_financial_account = Client.hasAccountType('financial', 1);
-                    const is_maltainvest = State.getResponse(`landing_company.mt_financial_company.${getMTFinancialAccountType(acc_type)}.shortcode`) === 'maltainvest';
-                    const is_financial = accounts_info[acc_type].account_type === 'financial';
-                    const is_demo = accounts_info[acc_type].account_type === 'demo';
+                    const is_maltainvest        = State.getResponse(`landing_company.mt_financial_company.${getMTFinancialAccountType(acc_type)}.shortcode`) === 'maltainvest';
+                    const is_financial          = accounts_info[acc_type].account_type === 'financial';
+                    const is_demo_financial     = accounts_info[acc_type].account_type === 'demo' && accounts_info[acc_type].mt5_account_type; // is not demo vol account
                     let is_ok = true;
 
-                    if (is_maltainvest && (is_financial || is_demo) && !has_financial_account) {
+                    if (is_maltainvest && (is_financial || is_demo_financial) && !has_financial_account) {
```

Comment:
what is the purpose of this check? isn't `is_financial` and `has_financial_account ` the same or what is the difference?

Reply:
I believe `is_financial` searches in accounts_info for mt5 for current account, and `has_financial_account` searches if user has MF account